modules/comments/types/comments.ts

```diff
@@ -35,7 +35,10 @@ export type CommentsAPIResponseItem = {
 };
 
 export type ParsedExecutiveComments = {
-  comment: Omit<ExecutiveComment, 'voterWeight'> & { voterWeight: BigNumber };
+  comment: Omit<ExecutiveComment, 'voterWeight'> & {
+    voterWeight: BigNumber;
+    gaslessNetwork?: SupportedNetworks;
```

Comment:
needed to add this property here to mantain the same type with the pollComments so the comment component does not complain, we will think about another approach in the future